js/functions.js

```diff
@@ -0,0 +1,33 @@
+const checkString = (testString, maxLength) => testString.length <= maxLength;
```

Comment:
Надо стремиться давать максимально говорящие названия, в данном случае подойдет лучше 
```suggestion
const checkStringLength = (testString, maxLength) => testString.length <= maxLength;
```